cabot/cabotapp/models.py

```diff
@@ -777,10 +793,11 @@ def description(self):
         help_text='Yaml representation of "header: regex" to match in '
                   'the results',
     )
-    timeout = models.IntegerField(
+    timeout = PositiveIntegerMaxField(
         default=30,
+        max_value=32,
```

Comment:
nit: can we put constants for max_value/default in a defs file (like https://github.com/Affirm/cabot/blob/master/cabot/metricsapp/defs.py)? Can make a new one in cabotapp/